pegasus/test/test_form_routes.rb

```diff
@@ -30,30 +30,58 @@ class FormRoutesTest < SequelTestCase
     end
   end
 
-  DEFAULT_DATA = {
-    email_s: 'fake@example.com',
-    name_s: 'fake_name',
-    experience_s: 'university_student_or_researcher',
-    location_s: 'somewhere',
-    location_flexibility_ss: ['onsite'],
-    description_s: 'description',
-    allow_contact_b: '1',
-    age_18_plus_b: '1',
-    email_preference_opt_in_s: 'yes'
-  }.freeze
-
-  def test_volunteer_engineer_submission_2015
-    stubs(:dashboard_user).returns(nil)
-    Pegasus.stubs(:logger).returns(nil)
-    stubs(:request).returns(stub(ip: '1.2.3.4'))
-    row = insert_or_upsert_form('VolunteerEngineerSubmission2015', DEFAULT_DATA.dup)
-    @form = Form.find(id: row[:id])
-
-    @form.update(processed_data: {location_p: '37.774929,-122.419416'}.to_json)
-
-    assert_equal 0.8236209090344097,
+  describe 'volunteer_engineer_submission_2015' do
+    before do
+      # Delete all volunteer sign-ups in the test database before each test
+      ::PEGASUS_DB[:forms].where(kind: 'VolunteerEngineerSubmission2015').delete
+
+      Pegasus.stubs(:logger)
+      stubs(:dashboard_user)
+      stubs(:request).returns(stub(ip: '1.2.3.4'))
+    end
+
+    it 'returns local results' do
+      create_volunteer name: 'Local Person', location: '37.774929,-122.419416'
+      results = search location: '37.774368,-122.428760'
+      assert_equal 0.8236209090344097, results.first['distance']
```

Comment:
Could you add an explanation for this number? A precise comparison to a float number like this seems really strict.

Reply:
I'm not sure actually - it's unchanged from the existing test that was here.  I think it's just a check that our location-based search gives consistent results.

Reply:
I added this test in 31a362001ddc3d81c9056bc9995dbc1fe2d11b02 (part of #22645) - it's just arbitrary (but valid) coordinates to ensure consistency of the API's distance calculation. I thought it was self-explanatory enough as 'fixture' data, but I wouldn't object if someone wanted to add a comment explanation to clarify further (or change the coordinates to something more easter-egg like).